apps/sms/style/sms.css

```diff
@@ -232,6 +231,15 @@ form[role="dialog"][data-type="edit"] .bb-tablist > li {
   pointer-events: none;
 }
 
+[data-type="list"] .summary {
```

Comment:
Since .summary only exists in thread list, having addtional ```.threadlist-item``` seems not that necessary?